src/buildblock/ProjDataInfo.cxx

```diff
@@ -288,7 +320,7 @@ ProjDataInfo::ProjDataInfo(const shared_ptr<Scanner>& scanner_ptr_v,
   max_tof_pos_num = 0;
   tof_increament_in_mm = 0.f;
   tof_mash_factor = 0;
-  num_tof_bins = 1;
+  num_tof_bins = 0;
```

Comment:
This makes nearly all ctests crash. In particular, `ProjDataInfoMemory` uses `get_num_timing_poss()` which is now inconsistent with `min/max_tof_pos_num` (which gives 1).

I do not understand why we need this.